### ISSUE TITLE
[Feature]: add friendly_name option

### ISSUE THREAD

**Issue by duczz** [mampfes/hacs_waste_collection_schedule#2110]
### I propose a feature for:

Sensors

### Describe your wanted feature

Can you add the friendly_name option for sensors?

**Comment by markvp** [comment]
Thanks for the request! The good news is this is already supported in a couple of ways.

**To set a custom name for a sensor**, use the \`name:\` key in the \`sensors:\` section of your \`configuration.yaml\`:

\`\`\`yaml
waste_collection_schedule:
  sensors:
    - name: My Custom Sensor Name
\`\`\`

**To rename the waste-type strings** that appear in sensor values (e.g. rename "green bin" to "Yard Waste"), use \`alias:\` under \`customize:\`:

\`\`\`yaml
waste_collection_schedule:
  sources:
    - name: your_source
      args: ...
      customize:
        - type: green bin
          alias: Yard Waste
\`\`\`

Full configuration reference: https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/configuration.yaml

Hope that helps — closing as already supported, but feel free to reopen if you had something different in mind!

**Comment by duczz** [comment]
Hey, via UI ist it not possible?

**Comment by markvp** [comment]
> Hey, via UI ist it not possible?

yes, when configuring you can use the Customize option to reconfigure the options, including `alias`.

https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/installation.md
